tests/dashbio_demos/app_manhattan_plot.py

```diff
@@ -18,7 +18,15 @@
 DATAPATH = os.path.join(".", "tests", "dashbio_demos", "sample_data", "manhattan_")
 
 # Load the data
-DATASET = pd.read_csv("{}data.csv".format(DATAPATH))
+DT = pd.read_csv("{}data.csv".format(DATAPATH))
+
+# Trim down the data
+if 'CHR' in DT:
+    datasets = [DT.loc[DT['CHR'] == i+1] for i in range(DT['CHR'].max())]
+    datasets = [dataset.iloc[:50] for dataset in datasets]
+    DATASET = pd.concat(datasets).reset_index(drop=True)
+else:
+    DATASET = None
```

Comment:
I don't think that's very satisfying... We want to return some kind of warning message at the very least. I would suggest replacing this if/else with either a try/except or `assert 'CHR' in DT`, in order to fire an error when `'CHR`' isn't available in the dataset (because then the demo app isn't usable anyway).

Reply:
We've done the same thing with other apps, since we know which datasets they will be using. (e.g., https://github.com/plotly/dash-bio/blob/51a88da4c10decf1421b63fc14bf4be21767ffd1/tests/dashbio_demos/app_clustergram.py#L67) I don't think it's actually necessary to do this sort of thing, since the warning message will never pop up (because, again, these apps were made with specific datasets in mind, and we can guarantee that that specific dataset has a column named `CHR`). I can do something similar to what I did with clustergram above, and define these values in a dictionary -- but either way, they'll end up hardcoded.